src/main/java/views/file_session_storage/FileSessionStorage.java

```diff
@@ -0,0 +1,47 @@
+package views.file_session_storage;
+
+import entities.SessionStorage;
```

Comment:
Only the use case layer (application business rules) should be accessing the entities package (Clean Architecture slides 04-page 12). We break that with the SessionStorage because there is no other way to access that information, so just make sure that for every class that you import it there is no other option (so you can explain it).

Reply:
OB, OD, G, MainMenuVB, and P all have it imported.

Reply:
Sorry, I meant only the G, MainMenuVB, and P since they are in the interface adapter layer! We could either make note of it in every Javadoc or state is elsewhere in our final submission? I am not sure we will likely need to discuss in group meeting.

Reply:
Okay. I'll wait for consensus before doing anything

Reply:
Sounds good!